src/test/java/Homework22.java

```diff
@@ -0,0 +1,20 @@
+
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+    public class Homework22 extends BaseTest {
+        public HomePage(WebDriver givenDriver) {
+            super(givenDriver);
+        }
```

Comment:
In the test script, you cant use Page Object Design Pattern -> import  `LoginPage `and `HomePage `class

//HINT
```
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.testng.annotations.Test;
import Pages.HomePage;
import Pages.LoginPage;

public class Homework22 extends BaseTest {

        @Test
        public void getUserAvatar() {
            LoginPage loginPage = new LoginPage(driver);
            HomePage homePage = new HomePage(driver);

            loginPage.login();
            homePage.getUserAvatar();
    }

}

```